src/app-layout/drawer/styles.scss

```diff
@@ -58,6 +58,9 @@
     block-size: 100%;
     position: relative;
   }
+  > .drawer-content-wrapper[aria-hidden='false'] {
```

Comment:
I did not find any condition when `aria-hidden=true` is applied on this selector. Does it mean we can remove that part of the selector?

Reply:
Interesting, that feels unexpected because `aria-hidden` is set to `!isOpen` [here](https://github.com/cloudscape-design/components/blob/main/src/app-layout/drawer/index.tsx#L124). I would've expected it to change when the drawer is opened/closed, like it does for help panel and side nav. 

It appears that `isOpen` always returns true because it is [hardcoded this way](https://github.com/cloudscape-design/components/blob/main/src/app-layout/classic.tsx#L550). Not sure exactly when or what changed to make the `isOpen` condition obsolete. But I have removed this condition from the style.